synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -141,21 +158,15 @@ public void onPerformSync(final @NonNull Account account, Bundle extras, String
 
```

Comment:
The do {} while {} content is definitelly too complex, I just sliced it a bit into prepare(), fill() and post() but we have to refactor this eventually.